core/src/main/scala/org/apache/spark/internal/config/Tests.scala

```diff
@@ -61,4 +61,19 @@ private[spark] object Tests {
     .version("3.0.0")
     .intConf
     .createWithDefault(2)
+
+  val RESOURCES_WARNING_TESTING = ConfigBuilder("spark.resources.warnings.testing")
+    .version("3.1.0")
+    .booleanConf
+    .createWithDefault(false)
+
+  // This configuration is used for unit tests to allow skipping the task cpus to cores validation
+  // to allow emulating standalone mode behavior while running in local mode. Standalone mode
+  // by default doesn't specify a number of executor cores, it just uses all the ones available
+  // on the host.
+  val SKIP_VALIDATE_CORES_TESTING =
+  ConfigBuilder("spark.testing.skipValidateCores")
+    .version("3.1.0")
+    .booleanConf
+    .createWithDefault(false)
```

Comment:
These 2 configs are backported from Master branch.

Reply:
ditto. This should be `3.0.1` when it comes to `branch-3.0`, @Ngone51 .
Also, after merging this, please update `master` branch consistently.

Reply:
Thank you @dongjoon-hyun for letting me know. I was wondering about it previously.